### PR TITLE
README.md: update

### PR DESCRIPTION
**Pull request by PushKK**
1. Small fix,
2. I not sure about fontconfig - needed or not.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -63,35 +63,35 @@ The following packages are **STRONGLY RECOMMENDED** to be installed:
...
  distributions. See http://pyicu.osafoundation.org/
- (These are Python bindings for the ICU package.
- https://pypi.python.org/pypi/PyICU/)
+ (These are Python bindings for the ICU package
+ https://pypi.python.org/pypi/PyICU/).
 
 * **Ghostscript**
 
-  Used by Graphviz reports to help create PDF's
+  Used by Graphviz reports to help create PDF's.
```
Comment (jralls) [inline README.md:71]: Since you're here please remove the apostrophe too.

**Comment by PushKK** [comment]
What you say about using fontconfig?
It's need for Gramps?

P.S. For Linux Mint 20 and 20.1 'fontconfig' not work for Gramps,
'python-fontconfig' not present more.

P.P.S. Can you select special symbol (U+2628 or U+2626) in .po file for each language?
1 way. Add to gramps.ini this rows:
```
[symbols]
dead=☦
```
And select symbols in Preferences.
2 way. Use gettext for symbol select:
`_("☦")
`In .po file user select need symbol for his language.

**Comment by SNoiraud** [comment]
See PR #1069

**Comment by PushKK** [comment]
Sorry, but 
```
>gramps/gen/utils/symbols.py

class Symbols(object):
    # genealogical symbols
    SYMBOL_FEMALE = 0
    SYMBOL_MALE = 1
    SYMBOL_ASEXUAL_SEXLESS = 2 # Unknown
...
    SYMBOL_ILLEGITIM = 9
```
not relation to ISO/IEC-5218-2004:
```
Data elements Code
Not known       0 (zero)
Male                1 (one)
Female            2 (two)
Not applicable 9 (nine)
```

**Comment by SNoiraud** [comment]
> not corresponds to ISO/IEC-5218-2004

Perhaps, but this is only for an internal use and  not to be exported for the moment.
The ISO norm you refer is really the  ISO/IEC DIS 5218 norm which is only in an inquiry stage.
So we can wait several years before it become a standard.

File: README.md [hunk 2]
```diff
@@ -102,7 +102,8 @@ The following packages are optional:
 * **ttf-freefont**
 
  More font support in the reports
-
+ For use genealogic symbols you must have FreeSerif or DejaVu Sans fonts (or other) installed.
```
Comment (Nick-Hall) [inline README.md:105]: This looks wrong. It gets appended to the previous line which doesn't end with a full stop.

Also DejaVu Sans is not included in this package.

**Review by Nick-Hall — CHANGES_REQUESTED** [review]
Apart from my one comment, everything else looks OK.

**Review by Nick-Hall — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (Nick-Hall) [reply README.md]: I changed this myself.

Comment (PushKK) [reply README.md]: 1. I check Gramps on fresh Linux system some years ago.
FreeSerif font had better MARRIAGE symbol - big and clear.
DejaVu - had this symbol too, but this symbol - small. 
2. Thank you.

**Pull request merged**